federation/cluster/common.sh

```diff
@@ -24,7 +24,19 @@
 
 : "${KUBE_ROOT?Must set KUBE_ROOT env var}"
 
+# Provides the $KUBERNETES_PROVIDER variable and detect-project function
+source "${KUBE_ROOT}/cluster/kube-util.sh"
+
+# Populates $PROJECT
+detect-project
+if [[ ${PROJECT} == *':'* ]]
```

Comment:
Shouldnt this also check that FEDERATION_PUSH_REPO_BASE is empty? Otherwise, I will always get this error if my project has ":" in it (which I think is true for @quinton-hoole's project)